docs/node/run-your-node/paratime-observer-node.mdx

```diff
@@ -0,0 +1,143 @@
+import DocCard from '@theme/DocCard';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# ParaTime Observer Node
+
+:::info
+
+These instructions are for setting up a _ParaTime observer_ node, which observes ParaTime activity and can issue
```

Comment:
Why not describe it by referring to client node ("is a special client node that supports confidential smart contact queries.").

Reply:
Sure, updated.